transit/models/__init__.py

```diff
@@ -0,0 +1,31 @@
+from transit.models.customer import (
```

Comment:
Just a details but we might consider using nested modules for some models. 
E.g.
Customer would be imported from models.customer and not models directly. Just a comment for future, in current application size it's perfectly fine.

Reply:
Yes, you are right. Good point. But anyway such imports are necessary to trigger 'makemigrations' command etc.